static/js/components/dashboard/CourseSubRow.js

```diff
@@ -1,4 +1,3 @@
-// @flow
```

Comment:
No, I ran into a bunch of flow errors when doing the other flow related changes. I'll see what files I can undo this change on